docs/examples/contrib/sqlalchemy/plugins/sqlalchemy_async_serialization_dto.py

```diff
@@ -8,7 +8,7 @@
 from litestar.plugins.sqlalchemy import SQLAlchemyDTO
 
 if TYPE_CHECKING:
-    from typing import List
+    pass
```

Comment:
I think we can just remove the entire block

Reply:
Yeah, no idea why ruff doesn't do these automatically?